uptime_service_validation/coordinator/server.py

```diff
@@ -97,10 +110,7 @@ def setUpValidatorPods(time_intervals, logging, worker_image, worker_tag):
                 name="CASSANDRA_PORT",
                 value=os.environ.get("CASSANDRA_PORT"),
             ),
-            client.V1EnvVar(
-                name="CASSANDRA_USE_SSL",
-                value="1"
-            ),
+            client.V1EnvVar(name="CASSANDRA_USE_SSL", value="1"),
```

Comment:
I liked previous formatting better. Its consistent with the other ones

Reply:
My vscode does this... :shrug: